src/tigerbeetle/cli.zig

```diff
@@ -287,6 +287,20 @@ const CLIArgs = union(enum) {
         verbose: bool = false,
     };
 
+    // Experimental: the interface is subject to change.
+    const Scrub = struct {
+        log_debug: bool = false,
+        seed: ?[]const u8 = null,
+        memory_lsm_manifest: ?ByteSize = null,
```

Comment:
I'd love to get rid of this parameter and fold it in, like how `storage_size_max` is determined from the actual data file size for the scrubber, but I'm not familiar enough with our node pool currently!